R/api_url.R

```diff
@@ -14,6 +14,7 @@
 #'
 #' @param endpoint Name of endpoint, can be "get-publications", "get-data-catalogue",
 #' "get-summary", "get-meta", "get-csv", "get-data" or "post-data"
+#' @param search String for filtering publication list
```

Comment:
Worth specifying that it filters by the publication title?

Reply:
Done